pulp_rpm/app/downloaders.py

```diff
@@ -18,12 +16,29 @@ def __init__(self, *args, **kwargs):
         """
         Initialize the downloader.
         """
+        self.sles_auth_token = None
+        self.silence_errors_for_response_status_codes = set()
+
         if 'sles_auth_token' in kwargs:
             self.sles_auth_token = kwargs.pop('sles_auth_token')
-        else:
-            self.sles_auth_token = None
+
+        if 'silence_errors_for_response_status_codes' in args:
```

Comment:
nitpick: I think these optional params would be better if they were in the __init__ signature above with a default to `None`. So that would be:

`def __init__(self, *args, sles_auth_token=None, silence_errors_for_response_status_codes=set(), **kwargs):`